index.html

```diff
@@ -5,15 +5,230 @@
+            </div>
+            <div class="main-date">
+                <h2>2022.1.15(MON) ~ 16(FRI)</h2>
+            </div>
+            <div class="main-center">
+                <p>@ United states of America, Science Center and more</p>
+            </div>
+        </section>
+    </header>
+    <section class="main-program" id="program">
+        <div class="heading-main d-flex flex-column align-items-center justify-content-center">
+            <h3>Main Program</h3>
+            <span class="col-3"></span>
+        </div>
+        <div class="main-main">
+            <div class="main-card card1">
+                <div class="lecture">         
+                    <i class="fa-solid fa-microphone px-3"></i>
+                    <h3>Lecture</h3>
+                </div>
+                <div class="main-paragraph ">
+                    <p>Listen to the speakers from various countries about the messages of sharing and opening.</p>
+                </div>
+            </div>
+            <div class="main-card card2">
+                <div class="lecture ">         
+                    <i class="fa-solid fa-palette"></i>
+                    <h3>CC Exhibition</h3>
+                </div>
+                <div class="main-paragraph ">
+                    <p>Appreciate various creations applying CC license of artists, organized from Art Center Nabi</p>
+                </div>
+            </div>
+            <div class="main-card card3">
+                <div class="lecture ">         
+                    <i class="fa-solid fa-comments"></i>
+                    <h3>Forum</h3>
+                </div>
+                <div class="main-paragraph ">
+                    <p>Have the time to share your thoughts and opinions with experts for each topic.</p>
+                </div>
+            </div>
+            <div class="main-card card4">
+                <div class="lecture ">         
+                    <i class="fa-solid fa-chalkboard"></i>
+                    <h3>Workshop</h3>
+                </div>
+                <div class="main-paragraph ">
+                    <p>Try creating your own work using open source license rather than just watching at it.</p>
+                </div>
+            </div>
+            <div class="main-card card5">
+                <div class="lecture ">         
+                    <i class="fa-solid fa-globe"></i>
+                    <h3>CC Ignite</h3>
+                </div>
+                <div class="main-paragraph ">
+                    <p>get opportunities to network with CC affiliates around the world, also after the summit.</p>
+                </div>
+            </div>
+        </div>
+        <div class="btn-container" id="join">
+            <button class="btn-main">Join CC Global Summit 2015</button>
+            <a href="#" class="btn-main-desk">SEE THE WHOLE PROGRAM</a>
+        </div>
+    </section>
+    <section class="speakers">
```

Comment:
This section is displaying content as expected and expands on mobile 👍💪✔, just fix the margin and add the missing divider also if the title is not provided don't display the empty quotes please do your best to replicate the give design 

![speakers section](https://user-images.githubusercontent.com/59707859/225582696-e55a3c10-2126-4456-9cd9-01ce448b773d.PNG)

![student speakers section](https://user-images.githubusercontent.com/59707859/225582709-67d02682-b4c8-4843-8369-ec7edbd2c9f2.PNG)